resources/js/composables/local-storage.js

```diff
@@ -0,0 +1,105 @@
+
+import { onMounted, reactive, ref, watch } from 'vue';
+
+
+
+function isRef(value) {
+    return (typeof value !== 'object');
+}
+
+function getRefValue(value) {
+    return isRef(value) ? ref(value) : reactive({ ...value });
+}
+
+function resetToDefault(storage, defaultValue) {
+    console.log(defaultValue);
```

Comment:
remove log

Reply:
Resolved in 5ae69a1a8ea093f394169744a5ed392dc4f666b8